index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta charset="UTF-8" />
+	<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+	<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+	<title>Survey Form</title>
+	<link rel="stylesheet" href="./style.css" />
+</head>
+
+<body>
+	<div class="container">
+		<h1 id="title">Survey</h1>
+		<p id="description">
```

Comment:
Kindly change your form name into more descriptive one. So that the users can understand what is the survey is about.